static/sass/_patterns_form-multiselect.scss

```diff
@@ -0,0 +1,94 @@
+@mixin snapcraft-p-multiselect {
+  .p-multiselect {
+    @extend %vf-clearfix;
+    align-content: flex-start;
+    align-items: flex-start;
+    border: 1px solid $color-mid-light;
+    border-radius: .125rem;
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, .12);
```

Comment:
This is the same as the current `box-shadow` for Vanilla text fields. But:
- If the Vanilla style changes in future, this will be left behind.
- It gets overridden anyway: http://snapcraft.io-pr-1089.run.demo.haus/static/css/styles.css has `p-multiselect, .p-multiselect .p-multiselect__options {box-shadow: 0 1px 5px 1px rgba(17, 17, 17, 0.2); }`, making these fields inconsistent with the rest of snapcraft.io.
![snapcraft io-1089-outset](https://user-images.githubusercontent.com/19801137/45538270-7e19db00-b7fe-11e8-8b11-b7e61a5fb7ad.png)